### PR TITLE
refactor(products): use asset-card & frame-button

### PR DESCRIPTION
**Pull request by luqven**
## Before this PR
The extension app did not use `AssetCard.tsx` or `FrameButton.tsx` components.

## After this PR
`frontend/src/components/ProductImage/ProductImageContainer.tsx` extension component uses these components.

## Screenshot 🖼️ 
ProductImageContainer

### CODE REVIEW

**Comment by luqven** [comment]
Current dependencies on/for this PR:
* next
  * **PR #235** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/235" target="_blank">Graphite</a> 
    * **PR #236** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/236" target="_blank">Graphite</a> 
      * **PR #237** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/237" target="_blank">Graphite</a> 
        * **PR #238** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/238" target="_blank">Graphite</a> 
          * **PR #239** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/239" target="_blank">Graphite</a> 
            * **PR #240** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/240" target="_blank">Graphite</a> 
              * **PR #241** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/241" target="_blank">Graphite</a> 
                * **PR #242** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/242" target="_blank">Graphite</a> 
                  * **PR #243** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/243" target="_blank">Graphite</a> 
                    * **PR #244** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/244" target="_blank">Graphite</a> 
                      * **PR #245** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/245" target="_blank">Graphite</a> 
                        * **PR #246** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/246" target="_blank">Graphite</a> 
                          * **PR #247** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/247" target="_blank">Graphite</a> 
                            * **PR #248** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/248" target="_blank">Graphite</a> 
                              * **PR #249** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/249" target="_blank">Graphite</a> 
                                * **PR #250** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/250" target="_blank">Graphite</a> 
                                  * **PR #251** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/251" target="_blank">Graphite</a> 
                                    * **PR #252** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/252" target="_blank">Graphite</a> 
                                      * **PR #253** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/253" target="_blank">Graphite</a>  (this PR)

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: frontend/src/components/ProductImage/ProductImageContainer.module.scss [hunk 1]
```diff
@@ -1,10 +1,38 @@
...
 .imageWrapper {
-  background-color: #e3e7eb;
-  height: 60px;
-  width: 60px;
+  background-color: $color_bg_new;
   overflow: hidden;
   display: flex;
   margin: 5px;
-  min-width: 60px;
+  max-width: 200px;
```
Comment (luqven) [inline frontend/src/components/ProductImage/ProductImageContainer.module.scss:8]: this helps keep the cards from going above the desired `200x200` dimensions while allowing the card to be smaller when needed.

File: frontend/src/components/ProductImage/ProductImageContainer.tsx [hunk 2]
```diff
@@ -25,21 +27,36 @@ export const ProductImageContainer = ({
...
+          <FrameButton size="small" color="primary" />
+          <p>Large</p>
+        </div>
+        <div className={styles.assetButtons}>
+          <FrameButton size="small" color="primary" />
+          <p>Medium</p>
+        </div>
+        <div className={styles.assetButtons}>
+          <FrameButton size="small" color="primary" />
+          <p>Small</p>
```
Comment (luqven) [inline frontend/src/components/ProductImage/ProductImageContainer.tsx:56]: S/M/L checkboxes. A future PR will properly style these as checkboxes.

**Comment by luqven** [comment]
> CI failures addressed in last PR

**Review by frederickfogerty — APPROVED** [review]

**Pull request merged**